indi-qhy/qhy_ccd.cpp

```diff
@@ -2185,6 +2232,15 @@ void QHYCCD::updateTemperature()
 
     CoolerN[0].value      = coolpower / 255.0 * 100;
     CoolerNP.s = CoolerN[0].value > 0 ? IPS_BUSY : IPS_IDLE;
+    if (hret == QHYCCD_SUCCESS)
```

Comment:
if (HasHumidity) should be moved here so all the code for it runs in a single block, i.e.

if (HasHumidity)
{
  double humidity=0;
  if (getQHYCCDHumidity(m_CameraHandle, &humidity) == QHYCCD_SUCCESS)
  {
    ....
  }
  else
  {
    ....
  }

  IDSetNumber(&HumidityNP, nullptr);

}

Reply:
done: 35ae0bb